IdentityCore/src/cache/mac/MSIDMacKeychainTokenCache.m

```diff
@@ -1249,6 +1253,8 @@ - (void) createError:(NSString*)message
     {
         *error = MSIDCreateError(domain, code, message, nil, nil, nil, context.correlationId, nil);
     }
+    
```

Comment:
should we do 
`else { return NO; }`
?

Reply:
Having nil error is a valid condition, so I'm concerned returning NO.